packages/cdk/lib/util/instance-types.ts

```diff
@@ -176,6 +176,10 @@ const unLaunchedInstanceTypesByRegion: { [key in string]: { [key in string]: boo
 
 export const getInstanceTypesForBatch = (instanceTypes: InstanceType[] | undefined, computeType: ComputeResourceType, region?: string): InstanceType[] => {
   if (instanceTypes && instanceTypes.length > 0) {
+    const armBased = instanceTypes.filter((instanceType) => isInstanceTypeArmBased(instanceType.toString()));
+    if (armBased) {
+      console.warn("Using ARM based instance type");
```

Comment:
1. nit: Better log message. (E.g. "ARM based instance type is not supported in Amazon Genomics CLI...")
2. Maybe see if this is a common issue with all ARM-based instances. If that's really the case, I think we should fail the process immediately instead of simply printing a warning message. According to #211, this cause problems down the road which might be confusing to customers if we let me deploy with ARM instances.

Reply:
Do we have any data for what engines/workflows actually run on ARM? I'd rather we fail early here if we know it doesn't work rather than warn them and still deploy all the resources only for it to fail later